static/css/around-map.css

```diff
@@ -2,10 +2,9 @@
 
 #open-form {
   position: absolute;
-  top: 0;
   left: 0;
   position: absolute;
-  top: 6rem;
+  bottom: 0rem;
```

Comment:
`bottom: 0;` Units are useless in that case

Reply:
addressed with 36763544bb3d5db9544dc83e263cc92dbc78c739